hammr/utils/migration_utils.py

```diff
@@ -29,3 +36,83 @@ def migration_table(migrations):
         table.add_row([migration.dbId, migration.name, status])
 
     return table
+
+
+def check_account_from_platform(accounts, credAccountName, targetPlatformName):
+    if accounts is None or len(accounts) == 0:
+        printer.out(
+            "No accounts available on the plateform.\n You can use the command 'hammr account create' to create an account.",
+            printer.ERROR)
+        return
+    else:
+        for account in accounts:
+            if account.name == credAccountName:
+                if account.targetPlatform.name == targetPlatformName:
+                    return account
+        return
+
+
+def check_migration_configuration(args_file):
+    file = generics_utils.get_file(args_file)
+    if file is None:
+        raise Exception("No such file or directory: " + args_file)
+    data = hammr_utils.load_data(file)
+
+    if "migration" in data:
+        migrationConfig = hammr_utils.check_mandatory_migration(data["migration"])
+        migrationConfig = hammr_utils.check_mandatory_migrate(migrationConfig)
+        if migrationConfig is None:
+            raise Exception("check yours parameters in file: " + args_file)
+        return migrationConfig
+    else:
+        raise Exception("no migration section found")
+
+
+def check_target(api, login, migrationConfig):
+    targetFormat = generate_utils.get_target_format_object(api, login,
+                                                           migrationConfig["destination"]["format"])
+    if targetFormat is None:
+        raise Exception("TargetFormat type unknown: " + migrationConfig["destination"]["format"])
+
+    targetPlatform = account_utils.get_target_platform_object(api, login,
+                                                              migrationConfig["destination"]["platform"])
+    if targetPlatform is None:
+        raise Exception("TargetPlatform type unknown: " + migrationConfig["destination"]["platform"])
+
+    return targetFormat, targetPlatform
+
+
+def check_publish_image(migrationConfig, targetFormat, file):
```

Comment:
naming is misleading, it retrieves the publish image.

Reply:
done